require.js

```diff
@@ -208,16 +204,14 @@
                         //     dependees[topId] = true;
                         // }
                         if((iPromise = deepLoad(normalizeId(resolve(depId, topId)), topId, loading))) {
-                            promises
-                                ? (promises.push ? promises.push(iPromise) : (promises = [promises,iPromise]))
-                                : (promises = iPromise);
+                            promises ? (promises.push ? promises.push(iPromise) : 
+                                (promises = [promises,iPromise])) : (promises = iPromise);
                         }
         			}
                 }
 
-                return promises
-                        ? (promises.push === void 0 ? promises : Promise.all(promises))
-                        : null;
+                return promises ? (promises.push === void 0 ? promises : 
+                            Promise.all(promises)) : null;
```

Comment:
jshint was complaining: Misleading line break before '?'; readers may interpret this as an expression boundary.